### ISSUE TITLE
"No module named 'pandas.core.index'" error when using pandas 2.0

### ISSUE THREAD

**Issue by mplpl** [IBM/nzpyida#18]
I have a new installation of nzpyida (on a new laptop) and so 'pip3 nzpyida' also installed the latest pandas which is 2.0.1.
Then when trying to import from nzpyida the following error is thrown:

```
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 from nzpyida import IdaDataBase, IdaDataFrame
      2 from nzpyida.analytics import DecisionTreeClassifier, AutoDeleteContext, ModelManager
      4 nzpy_cfg = {"user":"admin", 
      5     "password":"password", 
      6     "host":'9.30.57.160', 
   (...)
      9     "logLevel":0, 
     10     "securityLevel":1}

File [~/Documents/work/nzpyida_fork/nzpyida/__init__.py:19](https://file+.vscode-resource.vscode-cdn.net/Users/mpl/Documents/work/~/Documents/work/nzpyida_fork/nzpyida/__init__.py:19)
     17 standard_library.install_aliases()
     18 from .base import IdaDataBase
---> 19 from .frame import IdaDataFrame
     20 from .series import IdaSeries
     21 from .geoFrame import IdaGeoDataFrame

File [~/Documents/work/nzpyida_fork/nzpyida/frame.py:45](https://file+.vscode-resource.vscode-cdn.net/Users/mpl/Documents/work/~/Documents/work/nzpyida_fork/nzpyida/frame.py:45)
     43 import numpy as np
     44 import pandas as pd
---> 45 from pandas.core.index import Index, RangeIndex
     47 from lazy import lazy
     48 import six

ModuleNotFoundError: No module named 'pandas.core.index'
```

**Comment by mplpl** [comment]
This happens because pandas.core.index module was removed in pandas 2.0.0. We should get rid of importing it from frame.py.

**Comment by mplpl** [comment]
Fixed in 0.3.3
